example/thirdparty/arrowkt/build.mill

```diff
@@ -0,0 +1,72 @@
+import mill._
+import mill.scalalib._
+
+object Dependencies {
+  val kotlinVersion = "1.9.10"
+
+  // Compile dependencies
+  val arrowCore = ivy"io.arrow-kt:arrow-core:1.2.4"
+  val arrowFxCoroutines = ivy"io.arrow-kt:arrow-fx-coroutines:1.2.4"
+  val arrowResilience = ivy"io.arrow-kt:arrow-resilience:1.2.4"
+  val arrowFxStm = ivy"io.arrow-kt:arrow-fx-stm:1.2.4"
+  val arrowCollectors = ivy"io.arrow-kt:arrow-collectors:1.2.0"
```

Comment:
I'm not sure it should be like that. I guess you need to build these from the sources since they are the part of the repo.

The only external dependencies are declared here https://github.com/arrow-kt/arrow/blob/main/gradle/libs.versions.toml.

Reply:
Yes, these should be internal `moduleDeps`